hindi/art-of-living/who_developed_hindi.md

```diff
@@ -0,0 +1 @@
+Hindi (Hindi: हिन्दी hindī), sometimes spelled हिंदी, also called Modern Standard Hindi (Hindi: मानक हिन्दी mānak hindī), is a standardised and Sanskritised register of the Hindustani language. Hindi is one of the many official [[official languages |language]] of the Union of India.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.